code/game/objects/effects/weeds.dm

```diff
@@ -110,6 +110,10 @@
 /obj/effect/alien/weeds/weedwall/window
 	layer = ABOVE_TABLE_LAYER
 
+/obj/effect/alien/weeds/weedwall/window/Initialize()
+	. = ..()
+	RegisterSignal(src, COMSIG_MOB_SPECIAL_CLICK, .proc/special_click_signal)
+
```

Comment:
Register on init for the window under instead

Reply:
This doesn't seem to work; the atom you click is the weeds, which then fires the signal at the weeds.

Trying to then do say RegisterSignal(window, COMSIG_MOB_SPECIAL_CLICK, obj/structure/window_frame.proc/climb_or_whatever) on initialization goes runtime. If you really want, I'll tool around with it some more, maybe there was something missing, but I didn't have too much time to dick around and I figured the alternate solution I produced was fine